gtfs/spec/en/reference.md

```diff
@@ -29,174 +29,176 @@ This document explains the types of files that comprise a GTFS transit feed and
+|  stop_lat | Latitude | **Required** | Latitude of a location. | 
+|  stop_lon | Longitude | **Required** | Longitude of a location. |  
+|  zone_id | ID | **Conditionally Required** | Identifies the fare zone for a stop. This field is required if providing fare information using [fare_rules.txt](#fare_rulestxt), otherwise it is optional. If this `stop_id` represents a station or station entrance/exit, the `zone_id` is ignored. |  
+|  stop_url | URL | Optional | URL of a web page about a particular location. This should be different from the `agency_url` and the `route_url` field values. | 
+|  location_type | Enum | Optional | Indicates whether this `stop_id` represents a stop, station, or station entrance/exit. If `location_type` is omitted, all locations are treated as stops. Valid values are: |  
+|   |  | | `0` or "empty" - Stop. A location where riders board or disembark from a transit vehicle. | 
+|   |  | | `1`- Station. A physical structure or area that contains one or more stop. |  
+|   |  | | `2` - Station Entrance/Exit. A point where riders can enter or exit a station from the street. The record must also specify a parent station by setting `parent_station` equal to the relevant `stop_id`. | 
+|  parent_station | ID referencing `stops.stop_id` | Optional | Identifies the station that a stop or station entrance/exit is part of. This station-stop relationship, called a parent-child relationship, can only exist between a station (as the parent) and a stop or station entrance/exit (as the child). Recall that status as a station, stop or station entrance/exit is defined by `location_type`. If the stop or station identified in this record does not have a parent, `parent_station`="empty". Note that a station can never have a parent and a station entrance/exit must always have a parent. |  
+|  stop_timezone | Timezone | Optional | Timezone of the location. If a location has a parent station, it inherits the parent station’s timezone instead of applying its own. Stations and parentless stops with `stop_timezone`="empty" inherit the timezone specified by `agency.agency_timezone`. If `stop_timezone` values are provided, the times in [stop_times.txt](#stop_timetxt) should be entered as the time since midnight in the timezone specified by `agency.agency_timezone`. This ensures that the time values in a trip always increase over the course of a trip, regardless of which timezones the trip crosses. |
+|  wheelchair_boarding | Enum | Optional | Indicates whether wheelchair boardings are possible from the specified location. Valid values are: |
+|   |  | | For parentless stops: |
+|   |  | | `0` or "empty" - No accessibility information for the stop. |
+|   |  | | `1` - Some vehicles at this stop can be boarded by a rider in a wheelchair. | 
+|   |  | | `2` - Wheelchair boarding is not possible at this stop. |  
+|   |  | | For child stops: |  
```

Comment:
child stops? I would say for stops.

Reply:
"Child stops" means any stop that is associated with a station. A child stop will inherit the `wheelchair_boarding` behavior from its parent station if `wheelchair_boarding`=`0` or empty. On the other hand, for a parentless stop (stop not in a station), `wheelchair_boarding`=`0` or empty means no wheelchair accessibility information is available. So I think we need to include both kinds of stops separately.

Reply:
Yes ok, my bad.